cmd/mt-whisper-importer-reader/main.go

```diff
@@ -153,12 +165,26 @@ func processFromChan(files chan string, wg *sync.WaitGroup) {
 			req.Header.Add("Authorization", "Basic "+base64.StdEncoding.EncodeToString([]byte(*httpAuth)))
 		}
 
-		resp, err := client.Do(req)
-		if err != nil || resp.StatusCode != 200 {
-			log.Errorf("Error sending request to http endpoint %q: %q", *httpEndpoint, err)
+		success := false
+		attempts := 0
+		for !success {
+			pre := time.Now()
+			resp, err := client.Do(req)
+			passed := time.Now().Sub(pre).Seconds()
+			if err != nil || resp.StatusCode-(resp.StatusCode%100) != 200 {
```

Comment:
i think just `|| resp.StatusCode >= 300` is clearer

Reply:
fixed